src/wx/faudio.cpp

```diff
@@ -399,9 +393,8 @@ bool FAudio_Output::init(long sampleRate)
     if (gopts.upmix) {
         // set up stereo upmixing
         FAudioDeviceDetails dd;
-        ZeroMemory(&dd, sizeof(dd));
-        hr = FAudio_GetDeviceDetails(faud, 0, &dd);
-        assert(hr == S_OK);
+        memset(&dd, NULL, sizeof(dd));
```

Comment:
In C++, you can just value-initialize the struct to zero it out like this:
`FAudioDeviceDetails dd {};`